tests/terraform/checks/resource/aws/test_MWAAWebserverLogsEnabled.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+from pathlib import Path
+
+from checkov.terraform.runner import Runner
+from checkov.runner_filter import RunnerFilter
+from checkov.terraform.checks.resource.aws.MWAAWebserverLogsEnabled import check
+
+
+class MWAAWebserverLogsEnabled(unittest.TestCase):
```

Comment:
The convention is to add the prefix `Test` to the test classes so this should be `TestMWAAWebserverLogsEnabled`

Reply:
thanks :) i changed it